etc/daemons/pollerd.ini

```diff
@@ -24,14 +24,14 @@ daemon_enabled=1
 #-- SSL configuration --
 use_ssl=0
 # WARNING : Put full paths for certs
-#ca_cert=/etc/alignak/certs/ca.pem
-#server_cert=/etc/alignak/certs/server.cert
-#server_key=/etc/alignak/certs/server.key
+#ca_cert=/usr/local/etc/alignak/certs/ca.pem
+#server_cert=/usr/local/etc/alignak/certs/server.cert
+#server_key=/usr/local/etc/alignak/certs/server.key
 #hard_ssl_name_check=0
 
 #-- Local log management --
 # Enabled by default to ease troubleshooting
 use_local_log=1
-local_log=%(logdir)s/pollerd.log
+local_log=/usr/local/var/log/alignak/pollerd.log
```

Comment:
Those variables are not used anywhere by anyone ... some updated fields had some variables and some not. I choosed to include real values in all the fields rather than variables to make the files reflect what is the default configuration and because some tests (full_tst.py) and scripts (in dev) are using the default configuration of the repo.
I suggest to remain as of now ...